index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Flower Power Inc.</title>
+  
+  <link rel="stylesheet" href="style.css">
+  <script src="index.js" defer></script>
+      
+  <!-- Link to Google Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@300&family=Source+Serif+Pro:wght@200&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+
+  <header class="hero-image">
+    <div class="hero-text">
+      <h1>Flower Power Inc.</h1>
+      <h2>Fresh flowers at your doorstep!</h2>
+    </div>
+  </header>
+ 
+  <div class="intro">
+    <text>Do you love flowers but sometime do not have time to buy them? Flower Power Inc. brings you the nicest flowers in season picked by our star florists. The bouquets are delivered in a box outside your door. Read below to find out more. <strong>ENJOY!</strong> </text>
+  </div>
+
+  <main>
+
+ <!--Accordion-->
+
+  <div class="accordion">
```

Comment:
maybe try semantic elements like <section>, <article> instead of <div> as the technigo lectures suggested? I heard it helps web browser to understand html better :)